### ISSUE TITLE
System.Security.Cryptography.Xml 7.0.0 package doesn't work on .NET Framework

### ISSUE THREAD

**Issue by mconnew** [dotnet/runtime#79576]
### Description

When compiling against System.Security.Cryptography.Xml 7.0.0 for netstandard2.0 and running on .NET Framework, you get a TypeLoadException. This is because the net462 version of the assembly in the package is missing the type fowarding attributes to the .NET Framework assembly System.Sercurity.dll. It doesn't even reference System.Sercurity.dll.

### Reproduction Steps

Create a netstandard2,0 library that references the 7.0 version of the nuget package System.Security.Cryptography.Xml. Create a dummy class that references SignedXml in a static method (e.g. instantiates it). From a .NET Framework application, reference this library and call the static method. Compile and run.

### Expected behavior

Able to reference and use SignedXml in a .NET Standard library referenced from a .NET Framework application.

### Actual behavior

Throws the following exception:
```
System.TypeLoadException: Could not load type 'System.Security.Cryptography.Xml.SignedXml' from assembly 'System.Security.Cryptography.Xml, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
```

### Regression?

Yes, this works with the latest 6.0.x version (currently 6.0.1). In that version, the .NET Framework lib is for net461 (7.0.0 is for net462), it references System.Security.dll and includes all the necessary type forwarding for the various types.

### Known Workarounds

Reference the 6.0.1 version of the package.

**Comment by vcsjones** [comment]
This is duplicate of https://github.com/dotnet/runtime/issues/78652. It should be fixed in the 7.0.2 servicing release.

**Comment by mconnew** [comment]
I didn't find the original issue as I only looked for open issues. I would expect a problem that is currently still a problem as it hasn't been shipped yet to be findable in open issues as the set of open issues to dig through is a lot smaller than the set of closed issues. Closing this issue as it's a duplicate.
